.pre-commit-config.yaml

```diff
@@ -20,20 +20,4 @@ repos:
     rev: v2.3.0
     hooks:
       - id: trailing-whitespace
-      - id: check-yaml
       - id: end-of-file-fixer
-
-  - repo: local
```

Comment:
why this removal? if so make it as separate PR...

Reply:
We removed them as we decided not to use automatic formatting. So, we just keep the minimum. Will already merged it on master.

Reply:
I saw some camouflage commit with the removal but still why to remove it from the pre-commit, if you do not want to happy it, just do not use it... and still why is it part of this PR again?